lib/token-assert.js

```diff
@@ -96,11 +96,14 @@ TokenAssert.prototype.sameLine = function(options) {
 };
 
 /**
- * Requires tokens to be on different lines.
+ * Requires tokens to be on different lines. If atLeast, atMost, and exactly, aren't specified, defaults to at least 1.
  *
  * @param {Object} options.token
  * @param {Object} options.nextToken
  * @param {Object} [options.message]
+ * @param {Object} [options.atLeast] At least how many lines the tokens are apart
+ * @param {Object} [options.atMost] At most how many lines the tokens are apart
+ * @param {object} [options.exactly] Exactly how many lines the tokens are apart
```

Comment:
For reference, I am coming from a PR about disallowing multiple spaces and added in a similar `atMost` option. We noticed the discrepency and figured we should comment.

#1059